dev-libs/libsodium/libsodium-1.0.16.ebuild

```diff
@@ -37,7 +40,10 @@ src_configure() {
 		${myconf}
 }
 
-src_install() {
-	default
-	prune_libtool_files
+multilib_src_install() {
+	emake DESTDIR="${D}" install
+}
+
+multilib_src_install_all() {
```

Comment:
You probably want `einstalldocs` here.

Reply:
I've added it, works now locally for me.